qc_grader/grader/common.py

```diff
@@ -96,6 +97,12 @@ def pauliop_to_json(op: PauliOp) -> str:
 def noisemodel_to_json(noise_model: NoiseModel) -> str:
     return json.dumps(noise_model.to_dict(), cls=QObjEncoder)
 
+def SamplerResult_to_json(op: SamplerResult) -> str:
+    return json.dumps({
+        'metadata': op.metadata,
+        'coeff': op.quasi_dists
```

Comment:
would it make more sense labeling the field `quasi_dists` instead of `coeff`?

Reply:
I think it would. However that's the name they decided to use so that's why I tried to keep the labeling as it was.